themes/default/theme/src/scss/_search.scss

```diff
@@ -77,3 +77,9 @@
         @apply mx-auto justify-start;
     }
 }
+
+// Search box in left navigation (not in tabstrip)
```

Comment:
Not sure if this comment facilitates understanding what's going on. To someone without the context of this PR, this doesn't mean a lot.